doc/method/method.md

```diff
@@ -118,55 +121,42 @@ and either repaired manually (patched) or repaired by improvements to
 the heuristics.
 
 Alignment consists of scripted ad hoc patches followed by an automatic
-alignment procedure.  Scripted patches that allow source nodes to be
+alignment procedure.  Scripted patches allow source nodes to be
 correctly matched to union nodes in cases where this cannot be
-automatically inferred or where information might confuse the
-alignment process.  Scripted alignment patches include capitalization
-and spelling repairs, addition of synonyms (e.g. 'Heterokonta' as
-synonym for 'Stramenopiles'), name changes (e.g. 'Choanomonada' to
-'Choanoflagellida'), deletions (e.g. removing synonym 'Eucarya' for
-'Eukaryota' to avoid confusion with the genus in Magnoliopsida),
-removing paraphyletic taxa ('Protozoa', 'Protista').  [metrics on
-scripted alignments?  many non-alignment-related changes are also made
-at this point, but they would be better done in the final patch phase
-much later.]
-
-Automated alignment proceeds one source node at a time.  First, a list
-of candidate union matches, based on name, synonyms, and shared
-identifiers (if any), is prepared for the source node.  Then, a set of
-heuristics is applied to find a unique best union node match, if any,
-for that source node.
-
-Heuristics are of two kinds:
-
- 1. Separation heuristics are those that identify union nodes that
-    mustn't be match targets for the source node.
- 2. Preference heuristics are those that attempt to separate the set
-    of candidate targets into two groups, those that are more
-    appropriate vs. less appropriate as match targets for the given
-    source node.
-
-[This will have to be updated.  I tried this structure and it doesn't
-work.  The working version is very similar but not quite the same as
-this.]
-
-A heuristic is essentially a two-place predicate, applied to a source
-node and a union node, that either succeeds (the two could match
-according to this heuristic) or fails (they do not match according to
-this heuristic).  The method for applying the heuristics is as
-follows:
-
- 1. Start with a source node and a set C of union nodes (candidates).
+automatic, or prevent incorrect matches.  Scripted alignment patches
+include capitalization and spelling repairs, addition of synonyms
+(e.g. 'Heterokonta' as synonym for 'Stramenopiles'), name changes
+(e.g. 'Choanomonada' to 'Choanoflagellida'), deletions (e.g. removing
+synonym 'Eucarya' for 'Eukaryota' to avoid confusing eukaryotes with
+genus Eucarya in Magnoliopsida).
+
+There are NNN ad hoc patches to prepare for alignment.
+
+The automated alignment process proceeds one source node at a time.
+First, a list of candidate union matches, based on names and synonyms,
+is prepared for the source node.  Then, a set of heuristics is applied
```

Comment:
this is under the section 'candidate identification': 'we
limit the search to union nodes that have a name-string (either
primary or synonym) that matches any name-string of the source node.'
I thought that was pretty clear but maybe not?

I could clean up the text to set off the actual answer from the speculation, and/or put in a forward reference, or get rid of the section and move the information to where candidates are first mentioned.